app/src/main/java/com/github/se/orator/ui/friends/AddFriends.kt

```diff
@@ -130,7 +142,10 @@ fun AddFriendsScreen(
                             .height(AppDimensions.mediumHeight)
                             .focusRequester(focusRequester)
                             .testTag("addFriendSearchField"),
-                    label = { Text("Username", modifier = Modifier.testTag("searchFieldLabel")) },
+                    label = {
```

Comment:
I think it would be neater to import androidx.compose.material on top and keep the Text() declaration